assets/js/modules/analytics-4/components/common/PropertySelect.js

```diff
@@ -88,6 +88,37 @@ export default function PropertySelect( {
 	);
 
 	const { selectProperty } = useDispatch( MODULES_ANALYTICS_4 );
+
+	const measurementIDs = {};
+
+	useSelect( ( select ) => {
+		if ( ! properties.length ) {
+			return null;
+		}
+
+		const propertyMeasurementMapping = properties.map( ( property ) => {
+			const currentPropertyID = property._id;
+
+			const dataStream =
+				select( MODULES_ANALYTICS_4 ).getWebDataStreamsBatch(
+					currentPropertyID
+				);
+
+			if ( ! Object.keys( dataStream ).length ) {
+				isLoading = true;
+				return null;
+			}
+
+			const measurementID =
+				dataStream[ currentPropertyID ][ 0 ].webStreamData
+					.measurementId; // eslint-disable-line sitekit/acronym-case
+			isLoading = false;
+			return ( measurementIDs[ currentPropertyID ] = measurementID );
+		} );
+
+		return propertyMeasurementMapping;
```

Comment:
This doesn't do anything since you aren't utilizing the returned value. The `useSelect` hook just maps the `measurementId` to above-declared object `measurementIDs`.